phy/cluster/manual/clustering.py

```diff
@@ -6,15 +6,14 @@
 # Imports
 #------------------------------------------------------------------------------
 
-from collections import namedtuple, defaultdict, OrderedDict
-from copy import deepcopy
-
 import numpy as np
 
-from ...ext.six import iterkeys, itervalues, iteritems
 from ...utils.array import _as_array, _is_array_like
```

Comment:
Yeah, I also thought this was odd. I guess it makes a bit of sense if you want to refactor different bits into completely different independent modules, but it's much more logical to me to specify paths relative to the root so you can follow the folder structure!